packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -4,139 +4,105 @@ excerpt: >-
 
-* You have several OAuth applications that are split across multiple teams. You need to ensure that one OAuth application doesn't consume all of the assigned per minute request limits.
-* You want to isolate any rogue OAuth applications from other valid OAuth applications.
-* You want to ensure that every application team adheres to best-coding practices, such as proper error handling to prevent redirect loops.
-* You want protection against random synthetic tests or batch jobs in production that make numerous requests every minute.
+You can investigate high usage of a rate limit or a rate limit violation. Once you know the issue, you can take the appropriate corrective action. For example, if the issue is high traffic, you may want to increase the rate limits. In cases where the issue is a rogue app or bad script, you can disable the app, or block that traffic. Whatever the issue is, you can remediate the issue through self-service. In cases where a higher rate limit is required, you can create a support case, by sending a request to the correct team for a rate limit increase.
 
-### How it works
+## Access the dashboard
 
-The best way to describe how client-based rate limiting works is to provide some example use cases.
+When a rate limit violation occurs, you're notified in the following ways and can access the Rate Limits dashboard from the link in the notification:
 
-#### Client-based isolation for a unique IP address
+* You receive a rate limit email that informs you that you've hit a rate limit for an endpoint. Click **View utilization history** in the email to navigate to the Rate Limits Dashboard and System Log for that particular endpoint.
 
-Bob and Alice are working from home and have distinct IP addresses. Both Bob and Alice try to sign in to their company's portal application (clientID: portal123) at the same time. When they make the authorize request to `https://company.okta.com/oauth2/v1/default/authorize?clientId=portal123`, the client-based rate limit framework creates a unique per minute request quota for both Bob and Alice using their IP address and the OAuth client ID that they are trying to access.
+![Access the dashboard by email notification](/img/RLDashboard1.png "Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard")
 
-* Bob: (IP1 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
-* Alice: (IP2 + portal123) Gets a quota of 60 total requests per minute and a maximum of two concurrent requests
+* You've hit a rate limit when you're in the rate limits dashboard or the Admin console, and a banner displays to inform you. Click **utilization history** in the banner to open the Rate Limits Dashboard for that particular endpoint or **syslog** to open the System Log.
```

Comment:
```suggestion
* You've hit a rate limit when you're in the rate limits dashboard or the Admin Console, and a banner appears to inform you. Click **utilization history** in the banner to open the Rate Limits Dashboard for that particular endpoint or **syslog** to open the System Log.
```

Is it the Rate Limits Dashboard, the Rate Limits dashboard, or the rate limits dashboard?